csharp/nuget/NuGetPackageBuilder.ps1

```diff
@@ -0,0 +1,10 @@
+﻿$solutionRoot = Resolve-Path ..\
+
+# Delete any existing output.
+Remove-Item *.nupkg
+
+# Create new packages for any nuspec files that exist in this directory.
+Foreach ($nuspec in $(Get-Item *.nuspec))
+{
+	.\Tools\NuGet.exe pack "$nuspec"
```

Comment:
>NuGet [](start = 9, length = 5)

Just curious, is it common to commit this and nuget.exe to the repo? In the past, when I've created projects like this published on NuGet, I tend to create a separate (private) management repo with a submodule for the open source project.  That way I can keep private keys for code signing, scripts like this one, and NuGet.exe there.